### DISCUSSION TITLE
Postman Swags Poll

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/Opensource-practice#156] [Polls]
Did you receive Postman Swags?
Ideally, you should receive an email from Ali like this before your swag gets shipped. Have you received the same? Do the poll
![image](https://github.com/user-attachments/assets/2bafbc2a-e003-4682-a418-06186dc9f7fb)

![image](https://github.com/user-attachments/assets/321c4e9e-79f4-44d6-9c97-dd3831ee3b81)

**Comment by Madhurya2303** [comment]
I have received the mail but didn't get the swags.

**Reply by sanjay-kv** [reply]
you have received mail from ali which means postman team already started shipping it, mostly people do get goodies in a week. 
it depends on where you located as well. 
if you still don't receive swags after 1 week feel free to reach me out I will help you out.

**Reply by sanjay-kv** [reply]
I have meeting with ali now, will highlight the poll result to him

**Reply by kRiShNa-429407** [reply]
I also received mail and filled Google form but 15 days has been passed but  not received swags yet!

**Comment by Deekshashakyawal** [comment]
I have received the mail , and asked him to change the address, he asked me to fill the form again and I did so on 8 August, but I haven't received any update about the swags till now.

**Comment by kRiShNa-429407** [comment]
I filled the form about address, and everything but not received the swag yet

**Comment by SadafKausar2025** [comment]
Hello @sanjay-kv sir
In my shipment page from 2 days it is showing in transist and showing 7 sep expected date to recieve 
I hope I will recieve on 7th sep 

Is there any kvc is to do in shiprocket??

**Reply by Madhurya2303** [reply]
How to check shipment page

**Reply by sanjay-kv** [reply]
atm postman team wont let give more visibility on shipment as they have already mentioned some wiatiing time on the email u received. 
![image](https://github.com/user-attachments/assets/ce5d44e0-cf08-42e7-883a-09f7b568e232)

**Comment by hars-21** [comment]
I have received the mail and also filled the form for the address but haven't received the swags till now.

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/fdc95a78-c31b-45f7-89cd-5dc1fb361a71)
its take time, i hope you received the mail like this from postman. they wont take any question beyond this waiting period on shipments. ideallty people do get shipment in 2 to 3 weeks

**Reply by hars-21** [reply]
Yeah I received it like this only. Will wait for some more time

**Comment by Saksham2k3s** [comment]
Can someone please share the link to the shipment page or let me know how to check the status of swag shipments?
@SadafKausar2025

**Reply by SadafKausar2025** [reply]
Have you received mail from shiprocket??

**Reply by SadafKausar2025** [reply]
If yes then there is written track your order just click on that and check your delivery

**Reply by Saksham2k3s** [reply]
Not Yet!

**Comment by anwesha2002** [comment]
> Did you receive Postman Swags? Ideally, you should receive an email from Ali like this before your swag gets shipped. Have you received the same? Do the poll
> 
> ![image](https://private-user-images.githubusercontent.com/30715153/364372804-321c4e9e-79f4-44d6-9c97-dd3831ee3b81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU0NTk4NjYsIm5iZiI6MTcyNTQ1OTU2NiwicGF0aCI6Ii8zMDcxNTE1My8zNjQzNzI4MDQtMzIxYzRlOWUtNzlmNC00NGQ2LTljOTctZGQzODMxZWUzYjgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA0VDE0MTkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmM2FlMTAwYzQ4ZWNkZTZhZWQxMjZkYTZkMzgyNjgxOTEzYTc0OGEyNTNjMzNhMTVkYzNmOWVkZmUxNjliYWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dXeqYUBChNyLSwO9S_n52fzfjmixdTXrlPQ0O_f4Wbg)

@sanjay-kv sir, I have seen my name in the #batch5A winner list 1st row, in the linkedIn post made by gssoc. I got only mail from postman which is a little bit different from these mails samples that u have shared, it looks like this that's why I want to clarify my doubt
![Screenshot 2024-09-04 195031](https://github.com/user-attachments/assets/a7148c74-15b4-4577-bc97-7704e39f6866)

is this the mail i was supposed to be received or will I get another mail??
I received the mail on 31st aug

**Reply by sanjay-kv** [reply]
yes same

**Comment by AnkitMourya12** [comment]
@sanjay-kv  yes I received but the dairy and pen are missing in the swag

**Reply by sanjay-kv** [reply]
there were two promotion one for twitter, and LinkedIn.
if you linkedin winner you get diary +pen

**Reply by Madhurya2303** [reply]
@sanjay-kv I have posted on both but the link given to me is only Twitter 
Will I get the diary and pen ??

**Comment by Anand-Kumar7** [comment]
Hello Sanjay sir I have received mail from ali Mustafa and also the mail of shoprocket of swag delivered..
But I have not received any swags till now .

**Reply by sanjay-kv** [reply]
call any number available during shipping. 
check you gave proper address,
no one faced such issues before

**Comment by KomalSrivastava** [comment]
@sanjay-kv Hello sir,  I have received the mail but didn't get the swags.

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/1ba17944-b239-42da-9626-7ad0ca5f25cf)


read mails and instructions, its clearly mention it will take some time . after this period any issue you can raise issues.

**Comment by Deekshashakyawal** [comment]
@sanjay-kv Hello sir, I received the mail on 31st July , and asked to change the address and he said to fill the form again so I did , Its been a month since then but I haven't received any updates about swags till now.

**Reply by sanjay-kv** [reply]
as mentioned above 60 days is the wait period if that time is over I can ecslate to postman, else they wont take the ticket to solve the issue

**Comment by KomalSrivastava** [comment]
@sanjay-kv hello sir , i got email that my order is delivered but i haven't got anything yet. These are tha screenshot.. it is showing this what should I do now??

![Screenshot_20240907_191053](https://github.com/user-attachments/assets/6fd8cde6-42ba-45f1-90ab-bcb8510de4c5)
![Screenshot_20240907_190942](https://github.com/user-attachments/assets/6d084c27-0906-4120-8235-8e16582b2976)

**Reply by sanjay-kv** [reply]
@KomalSrivastava have you recieved your order, if not have you given proper contact details and address?

**Reply by KomalSrivastava** [reply]
@sanjay-kv I haven't received my swag but it is showing delivered.. I have given proper details in the form but still this issue occurred i don't know whyy🥲 what's the solution??

**Reply by sanjay-kv** [reply]
just wondering why everything going wrong, everyone else receiving swags properly, any contact number to reach out to them, 
as the delivery is handled by postman we have no details about shipment or tracking

**Comment by saketh-05** [comment]
@sanjay-kv I have received swag mail and filled out Google form but till now did not receive swags or any other mail regarding shipment details. I did not receive any email from ship rocket. Please help me 🙏

**Reply by sanjay-kv** [reply]
![image](https://github.com/user-attachments/assets/d56f3085-145e-4bd6-94a6-998131ee8782)


i mean its mentioned some waiting period.

**Reply by sanjay-kv** [reply]
@saketh-05 
Bro why not reading mails and attached picture above as well and ask same qn how will i be able to answer your question you already have a tracker , it has mentioned waiting time, some people get swags faster depends upon where they located. 
it is handled by postman shipping team again its a 3rd party shipment company. 

They take some time to handle and to reduce the query coming they set a deadline of 40 to 60 days which is communicated on swag fill form , and emails. and it has posted multiple time on the above discussion. 

Its just 10 days u filled the form, it takes time. that's why the waiting period for a reason. Thank god you didn't reach out to postman team directly what they do is instant cancellation of swags saying you didn't follow terms and conditions. 

![image](https://github.com/user-attachments/assets/2d342a78-9f77-4d34-9970-462afcc28b5f)

**Reply by saketh-05** [reply]
@sanjay-kv I apologise for the inconvenience caused. My bad I didn't understood correctly. Thank you for replying patiently. I will wait.

**Comment by saumyasri123** [comment]
Hello @sanjay-kv  Sir, I haven't received the swags yet. Please look into this. It's been 10 days of polling.

**Reply by sanjay-kv** [reply]
mate  read the mails and messages, 

![image](https://github.com/user-attachments/assets/a6eab44e-eaf3-453b-b419-9d46c53445c0)

**Comment by shoriful-dev** [comment]
Anyone here please mark me.

**Reply by Anand-Kumar7** [reply]
write proper message on what you have to mark
